internal/ingress/controller/nginx.go

```diff
@@ -673,6 +677,11 @@ Error: %v
 //
 //nolint:gocritic // the cfg shouldn't be changed, and shouldn't be mutated by other processes while being rendered.
 func (n *NGINXController) OnUpdate(ingressCfg ingress.Configuration) error {
+	workerSerialReloads := n.store.GetBackendConfiguration().WorkerSerialReloads
+	if workerSerialReloads && n.workersReloading {
+		return errors.New("worker reload already in progress, requeuing reload")
+	}
+
 	cfg := n.store.GetBackendConfiguration()
```

Comment:
We can just use the `cfg` instead of call it twice.

Reply:
done :)